src/main/java/com/banking/app/bankingApp/controller/UserController.java

```diff
@@ -0,0 +1,56 @@
+package com.banking.app.bankingApp.controller;
+
+import com.banking.app.bankingApp.Users.CreateUser;
+import com.banking.app.bankingApp.Users.UpdateUser;
+import com.banking.app.bankingApp.Users.User;
+import com.banking.app.bankingApp.service.users.UserManagementService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class UserController {
+    private UserManagementService userManagementService;
+    public UserController(){
+        userManagementService = new UserManagementService();
+    }
+    @PostMapping(value="/users")
+    public ResponseEntity<CreateUser> addUser(@RequestBody CreateUser createUser){
+        userManagementService.addUser(createUser);
+        return ResponseEntity.status(HttpStatus.CREATED).body(createUser);
```

Comment:
Return User here. Any time you return a user resource, you should use common User model.